### PR TITLE
sol1

### PR DESCRIPTION
**Pull request by Unlie9**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,9 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, file_name, direction = command.split()
+    dir_ = direction
```
Comment (Oleg-s-s) [inline app/main.py:6]: Why you create additional var?

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]

File: file.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This is some
+ content for
+ the file.
```
Comment (DevSpaciX) [inline file.txt:3]: Do not push redundat files into Pull reqest : )

**Review by DevSpaciX — CHANGES_REQUESTED** [review]

File: file.txt (hunk 2 above)
Comment (Unlie9) [reply file.txt:3]: XD

File: app/main.py [hunk 3]
```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, file_name, direction = command.split()
+    dir_ = os.path.dirname(direction)
+
+    if dir_:
+        os.makedirs(dir_, exist_ok=True)
```
Comment (DevSpaciX) [inline app/main.py:9]: You could use here operator warlus operator 

```
if dir_ := os.path.dirname(direction):
    os.makedirs(dir_, exist_ok=True)
```

That way you code will be a bit cleaner : )

**Review by DevSpaciX — CHANGES_REQUESTED** [review]
[Read about warlus](https://realpython.com/python-walrus-operator/) operator that is really useful

**Review by DevSpaciX — APPROVED** [review]
Perfect !

**Pull request closed without merging**